sdcm/utils/common.py

```diff
@@ -312,6 +313,18 @@ def set_public_access(self, key):
         acl_obj.put(ACL='', AccessControlPolicy={'Grants': grants, 'Owner': acl_obj.owner})
 
     def download_file(self, link, dst_dir):
+        """Download file from S3 bucket or Argus proxy link."""
+
+        if not self.s3_host_name_regex.match(urlparse(link).hostname):
```

Comment:
we should rather match for 'argus' domain - otherwise we could leak credentials if someone passed malicious host.

Reply:
also I don't want to limit it to specific URL of argus, it could limit us in testing with staging env and such.

Reply:
anyhow it's blocking people from restoring monitoring (and fail CI provisioning test), I'm pushing it in.